data/cybox/nozomi/nozomi_alerts_22022024.json

```diff
@@ -0,0 +1,563 @@
+{
+    "type": "bundle",
+    "id": "bundle--9e7bdaca-d33f-4b5f-88be-45420adb8bf2",
+    "objects": [
+        {
+            "type": "identity",
+            "id": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "name": "Nozomi",
+            "identity_class": "events",
+            "created": "2023-04-11T16:11:11.878Z",
+            "modified": "2023-04-11T16:11:11.878Z"
+        },
+        {
+            "id": "observed-data--974f9397-2340-4009-8fb8-94536ccd97cd",
+            "type": "observed-data",
+            "created_by_ref": "identity--f431f809-377b-45e0-aa1c-6a4751cae5ff",
+            "created": "2024-02-22T06:51:20.235Z",
+            "modified": "2024-02-22T06:51:20.235Z",
+            "objects": {
+                "0": {
+                    "type": "x-ibm-finding",
+                    "alert_id": "12ee457d-ca1c-4d98-9430-96b56afc9f76",
+                    "x_is_acknowledged": false,
+                    "severity": 80,
+                    "time_observed": "2023-12-12T07:10:57.012Z",
+                    "src_ip_ref": "1",
+                    "x_alert_status": "open",
+                    "name": "Sigma rule match",
+                    "x_cause": "Rule-dependent. A suspicious local event has been detected on a machine.",
+                    "ioc_refs": [
+                        "10"
+                    ],
+                    "x_solution": "Rule-dependent. Verify the device configuration and status, and the possible presence of malicious processes.",
+                    "description": "The following suspicious local event was detected on this host: \"Detects the execution of renamed PuTTY Plink to perform data exfiltration through tunnel port forwarding\".",
+                    "x_is_incident_alert": false,
+                    "x_is_cybersecurity_alert": true,
+                    "finding_type": "alert",
+                    "start": "2023-12-12T07:10:57.012Z",
+                    "rule_names": [
+                        "sigma_rules"
+                    ],
+                    "x_sensor_host": "EC2AMAZ-XXXXX",
+                    "x_alert_type_id": "SIGN:SIGMA-RULE",
+                    "x_rule_id": "5f9b24dd-fa05-401c-ad21-240b3f761c9f"
+                },
+                "1": {
+                    "type": "ipv4-addr",
+                    "value": "1.1.1.1",
+                    "x_nozomi_info_ref": "3"
+                },
+                "2": {
+                    "type": "network-traffic",
+                    "src_ref": "1",
+                    "protocols": [
+                        "unknown"
+                    ]
+                },
+                "3": {
+                    "type": "x-nozomi-info",
+                    "label": "EC2AMAZ-XXXXX",
```

Comment:
This seems to be a partially obscured real hostname. When obscuring values, it's best to remove any references to the real data and use a defined replacement. For example "Hostname" or "Host" might work better.